res/res_pjsip_refer.c

```diff
@@ -733,6 +736,49 @@ static void refer_blind_callback(struct ast_channel *chan, struct transfer_chann
 
 	pbx_builtin_setvar_helper(chan, "SIPREFERRINGCONTEXT", S_OR(refer->context, NULL));
 
+	get_xfrdata = pbx_builtin_getvar_helper(chan, "GET_TRANSFERRER_DATA");
+	if (!ast_strlen_zero(get_xfrdata)) {
+		const pjsip_msg * msg = refer->rdata->msg_info.msg;
+		const struct pjsip_hdr *end = &msg->hdr;
+		struct pjsip_hdr *hdr = end->next;
+		struct ast_str *pbxvar = ast_str_alloca(MAX_HDR_SIZE + 1);
```

Comment:
Why the use of ast_str_alloca over char pbxvar[MAX_HDR_SIZE + 1]; ? A comment should also be placed by each for why + 1

Reply:
Good point - rewriten in new version.